### ISSUE TITLE
graphql-tag-pluck config is broken

### ISSUE THREAD

**Issue by JimmyPaolini** [B2o5T/graphql-eslint#1173]
### Issue workflow progress

_Progress of the issue based on the [Contributor Workflow](https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md#a-typical-contributor-workflow)_

- [ ] 1. The issue provides a reproduction available on GitHub, Stackblitz or CodeSandbox
     > Please make sure the graphql-eslint version under `package.json` matches yours.
- [ ] 2. A failing test has been provided
- [ ] 3. A local solution has been provided
- [ ] 4. A pull request is pending review

---

**Describe the bug**

`graphql-tag-pluck` options are not being loaded into the plugin.

**To Reproduce**

Define a file `.graphqlrc.js`:
```
module.exports = {
  operations: './src/**/*.{tsx, ts, jsx, js}',
  documents: './src/**/*.{tsx, ts, jsx, js}',
  default: {
    extensions: {
      graphqlTagPluck: {
        modules: [{ name: 'graphql/tags', identifier: 'custom' }],
        globalGqlIdentifierName: 'custom',
        gqlMagicComment: '__nomagiccomment__',
      },
    },
  },
}
```
and `.eslintrc.js` overrides:
```
    {
      files: ['src/**/*.{js,jsx,ts,tsx}'],
      processor: '@graphql-eslint/graphql',
    },
    {
      files: ['*.graphql'],
      parser: '@graphql-eslint/eslint-plugin',
      plugins: ['@graphql-eslint'],
      rules: {
        '@graphql-eslint/unique-operation-name': 'error',
      },
    },
```

And any graphql queries/mutations created with the `custom` tag will not be linted, even though graphql-tag-pluck is configured.

**Expected behavior**

The `custom` GraphQL tag is loaded properly and linted

**Environment:**

- OS: macOS Monterey 12.6
- `@graphql-eslint/eslint-plugin`: `^3.11.2`
- Node.js: `v16.14.0`

**Additional context**

The graphql-tag-pluck extension config in the graphql-config is being overwritten here: https://github.com/B2o5T/graphql-eslint/blob/master/packages/plugin/src/graphql-config.ts#L21

This test is now insufficient in how it mocks the graphql-config loading: https://github.com/B2o5T/graphql-eslint/blob/master/packages/plugin/tests/processor-with-graphql-config.spec.ts#L3

**Comment by B2o5T** [comment]
Could you test `3.11.3-alpha-20221008220048-6ed3933` and say if your issue was fixed?

also, your graphql config is invalid refactor it like this, and I renamed `graphqlTagPluck` to `pluckConfig`

```diff
module.exports = {
- operations: './src/**/*.{tsx, ts, jsx, js}', // there is no operations field in graphql-config
  documents: './src/**/*.{tsx, ts, jsx, js}',
- default: {
    extensions: {
-     graphqlTagPluck: {
+     pluckConfig: {
        modules: [{ name: 'graphql/tags', identifier: 'custom' }],
        globalGqlIdentifierName: 'custom',
        gqlMagicComment: '__nomagiccomment__',
      },
    },
- },
}
```

**Comment by B2o5T** [comment]
should be fixed in `@graphql-eslint/eslint-plugin@3.12.0`

**Comment by JimmyPaolini** [comment]
Yup it now properly loads my graphql-tag-pluck config, thank you!
